lib/hideUI.js

```diff
@@ -12,10 +12,23 @@ define(function (require, exports, module) {
 	 * for bramble to be loaded
 	 */
 	function hide() {
-		removeMainToolBar();
-		removeLeftSideToolBar();
-		removeRightSideToolBar();
-		removeTitleBar();
+		if(!(_parseParams("ui") === "1")) {
+			removeMainToolBar();
+			removeLeftSideToolBar();
+			removeRightSideToolBar();
+			removeTitleBar();
+		}
+	}
+
+	/**
+	 * This function parses the current url for brackets, and looks for Get Methods
+	 */
+	function _parseParams(sParam) {
+		var sPageURL = window.location.search.substring(1);
```

Comment:
So I'd rename this to `function shouldHideUI()` and have it return `true` or `false` depending on whether that param is set to `1` or not.